### ISSUE TITLE
Create Accessibility Statement + Supporting Docs

### ISSUE THREAD

**Issue by logancribbs** [lsc-compsci/WCVS-Veteran_e-Post_Hub#63]
Need to write up a formal statement that says the website meets ADA compliance. Should have a PDF document of all supporting docs, such as screenshots, examples, and testing metrics for confirmation of WCAG compliance.
